### PR TITLE
option to discard test suite (#1)

### PR DESCRIPTION
**Pull request by DavidMarec**
Append the option 
`-DRUN_TEST_SUITE=OFF`

to **Cmake** command line to discard the test suite run.

This is required to allow cross compilation.

### CODE REVIEW

**Comment by DavidMarec** [comment]
This option should be integrated into gmmlib also

**Comment by mypopydev** [comment]
LGTM, I don't think always run unit test is a good idea.

**Comment by DavidMarec** [comment]
@mypopydev 
The main idea behind this proposal is to allow cross compilation.
This is a patch I am currently using to make Yocto build the driver.

**Comment by oliver-sang** [comment]
@DavidMarec could you update your PR according to my review? Or respond to my review comments if you have other ideas? Thanks

**Comment by DavidMarec** [comment]
@oliver-sang 
I 'd be glad to do so, but
I may have missed something: where to find your review ?

**Comment by oliver-sang** [comment]
@DavidMarec sorry, didn't 'submit' the review

File: CMakeLists.txt [hunk 1]
```diff
@@ -46,6 +46,8 @@ install (FILES ${CMAKE_CURRENT_BINARY_DIR}/media_driver/iHD_drv_video.so DESTINA
 install (FILES ${CMAKE_CURRENT_BINARY_DIR}/cmrtlib/linux/igfxcmrt64.so DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT media)
 
 option (INSTALL_DRIVER_SYSCONF "Install driver system configuration file" ON)
+option (RUN_TEST_SUITE "run google test module after install" ON)
```
[the remark this answers is not in the document]
Comment (DavidMarec) [reply CMakeLists.txt]: Doing so will not populate the gmmlib makefile, which needs this option too for similar reasons

**Pull request merged**